docs/Developers/Newcomers_Guide.rst

```diff
@@ -338,14 +338,15 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
     **Do not only fix a newcomer issue!** It is highly recommended that you
     fix one newcomer issue to get familiar with the workflow at coala and
-    then proceed to a difficulty/low issues.
+    then proceed to a ``difficulty/low`` issue.
 
     However those who are familiar with opensource can start with
-    "difficulty/low" issues.
+    ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's "difficulty/low" or
-    newcomer issues after you complete your newcomer issue as reviewing helps,
-    you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
```

Comment:
> it would have been too much work than...making more commits and than squashing tham in to one

You wouldn't be squashing the commits into one, the two commits would be kept separate. 

> my commit passes all test ..do u really want to make seperate commit for tht

I know it seems a bit tedious, but it'll make work easier if particular changes are referenced for future contributors; not in this particular case since it's documentation - but for code in general, you should make changes specific to what the commit says it does :)